client/src/components/utils/Login/index.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
```

Comment:
I think login is not utils  :)

Reply:
Yes, you're right. I'll create a new folder called "pages" and put all pages components inside it.